src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: normal;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  margin: 20px 10px;
+
+}
+
+.search {
+  width: 100%;
+  color: #3d4e61;
+  font-family: "Anevir", Arial, sans-serif;
```

Comment:
```suggestion
```
No need to specify your font-family here if you have already written it in `html` selector